mis_builder/models/expression_evaluator.py

```diff
@@ -1,11 +1,16 @@
 # Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
+import logging
+
 from .mis_safe_eval import NameDataError, mis_safe_eval
 
+_logger = logging.getLogger(__name__)
+
 try:
     import itertools.izip as zip
 except ImportError:
+    _logger.debug("Itertools not imported", exc_info=1)
```

Comment:
Actually, logging this is misleading, because `zip` is the python3 builtin equivalent of izip.
So I'd simply remove this python2 compatibility block.